dokku

```diff
@@ -195,7 +195,7 @@ case "$1" in
     done < "$DOKKU_SCALE_FILE"
 
     dokku_log_info1 "Running post-deploy"
-    plugn trigger post-deploy $APP $port $ipaddr
+    plugn trigger post-deploy $APP $port $ipaddr $IMAGE_TAG
```

Comment:
Oh I see. This would break any `pre-deploy` plugins that used those arguments. While I agree with you that this should probably happen at some point, I don't think it should go in this PR.